WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -19,8 +19,17 @@ class EditCommentTableViewController: UITableViewController {
     // So save and use one instance of the cell.
     private let commentContentCell = EditCommentMultiLineCell.loadFromNib()
 
+    // A closure executed when the view is dismissed.
+    // Returns the Comment object and a Bool indicating if the Comment has been changed.
+    @objc var completion: ((Comment, Bool) -> Void)?
```

Comment:
Since the Comment object is passed by reference, do we need to pass back the instance in the completion block? Any changes made to the Comment object should also be applied to the sender since it's the same instance.

Reply:
Um, yea, I suppose. But IMO it's not very obvious and may be a bit confusing. And it feels safer to explicitly pass the Comment rather than assuming the caller's behavior. If you feel strongly about it, I can change it later. This just feels better to me. 😄